lua/astrocommunity/media/image-nvim/README.md

```diff
@@ -0,0 +1,22 @@
+# image.nvim
+
+This plugin attempts to add image support to Neovim.
+
+It works wonderfully with Kitty + Tmux, and it handles painful things like rendering an image at a given position in a buffer, scrolling, windows, etc.
+
+It has built-in Markdown and Neorg integrations that you can use right now.
+It can also render image files as images when opened.
+
+Requirements
+These are things you have to setup on your own:
+
+ImageMagick - mandatory
+magick LuaRock - mandatory (luarocks --local install magick or through your package manager)
+Kitty >= 28.0 - for the kitty backend
+ueberzugpp - for the ueberzug backend
+curl - for remote images
+
```

Comment:
The format of the requirements should follow other README's with similar content

Reply:
Yes sorry, my bad.

Formatted the readme.
Hope it is ok now.